src/components/Note/Note.jsx

```diff
@@ -2,31 +2,110 @@ import React from "react";
+    } else {
+      notesDispatch({ type: "UNARCHIVE_NOTE", payload: singleNote });
+    }
+  };
+
   return (
     <div className={`card m-1 p-2 note-color-${noteColor}`}>
       <div className="note-header">
-        <h3>{title}</h3>
-        {pinned ? (
-          <div className="note-icon center-div" title="Unpin note" onClick={pinNoteHandler}>
-            <i className="fas fa-thumbtack pin-note-icon"></i>
-          </div>
-        ) : (
-          <div className="note-icon center-div" title="Pin note" onClick={pinNoteHandler}>
-            <i className="fas fa-thumbtack unpin-note-icon"></i>
-          </div>
-        )}
+        <h3 className="note-title">{title}</h3>
+        {!trash &&
+          !archive &&
+          (pinned ? (
+            <div
+              className="note-icon center-div"
+              title="Unpin note"
+              onClick={pinNoteHandler}
+            >
+              <i className="fas fa-thumbtack pin-note-icon"></i>
+            </div>
+          ) : (
+            <div
+              className="note-icon center-div"
+              title="Pin note"
+              onClick={pinNoteHandler}
+            >
+              <i className="fas fa-thumbtack unpin-note-icon"></i>
+            </div>
+          ))}
       </div>
       <div className="note-main">
         <p>{note}</p>
       </div>
-      <div className="note-footer"></div>
+      <div className="note-footer py-2">
+        {trash ? (
+          <div className="note-footer-options">
+            <div
+              className="note-icon center-div"
+              title="Delete forever"
+              onClick={() =>
+                notesDispatch({ type: "DELETE_FROM_TRASH", payload: _id })
+              }
+            >
+              <i className="fas fa-trash-alt"></i>
+            </div>
+            <div
+              className="note-icon center-div"
+              title="Restore"
+              onClick={() =>
+                notesDispatch({
+                  type: "RESTORE_FROM_TRASH",
+                  payload: singleNote,
+                })
+              }
+            >
+              <i class="fas fa-trash-restore"></i>
+            </div>
+          </div>
+        ) : (
+          <>
+            <small className="note-date">Created at</small>
+            <div className="note-footer-options">
+              {archive ? (
+                <div
+                  className="note-icon center-div"
+                  title="Unarchive"
+                  onClick={() => archiveNoteHandler("unarchive")}
+                >
+                  <i className="fas fa-archive"></i>
+                </div>
+              ) : (
+                <div
+                  className="note-icon center-div"
+                  title="Archive"
+                  onClick={() => archiveNoteHandler("archive")}
+                >
+                  <i className="fas fa-archive"></i>
+                </div>
+              )}
```

Comment:
Suggestion:
Please do ternary condition for title attribute instead of doing for entire element.

Reply:
Thank you for the suggestion. i will make the changes.